### PR TITLE
Support for Psiphon test

### LINKED ISSUE #332: Present findings in Explorer for Psiphon
Depends on: https://github.com/ooni/pipeline/issues/234

### PR DESCRIPTION
**Pull request by sarathms**
Fixes #332

### CODE REVIEW

**Comment by sarathms** [comment]
image

@agrabeli Can you please add strings in copy for the summary text and the hint under the status in Hero? (Rows 152-157 in the [spreadsheet](https://docs.google.com/spreadsheets/d/1QAeDVF5wYj7bnTE6rJ0XX3C8sBF0o6A0TxeLw9fpTvM/edit#gid=0)

**Comment by hellais** [comment]
@sarathms I added strings for these in the spreadsheet. I say you go ahead with those and then @agrabeli can perhaps review them as part of some future iteration.

File: components/test-info.js [hunk 1]
```diff
@@ -112,6 +113,11 @@ export const testNames = {
     name: <FormattedMessage id='Tests.BridgeReachability.Name' />,
     info: 'https://ooni.io/nettest/tor-bridge-reachability/'
   },
+  'psiphon': {
+    group: 'circumvention',
+    name: <FormattedMessage id='Tests.Psiphon.Name' />,
+    info: 'https://ooni.io/nettest/psiphon/'
```
Comment (hellais) [inline components/test-info.js]: This should link to `ooni.org`

**Comment by sarathms** [comment]
> I would suggest we use the Anomaly hero

Done.

I have gone ahead and updated all instances of `ooni.io` to `ooni.org` across the app.

**Pull request merged**